### ISSUE TITLE
Are ERC20 tokens supported?

### ISSUE THREAD

**Issue by appsdev9** [InfraexDev/peatio#112]
Hi,

Does InfraexDe_peatio support ERC20 tokens? 

Also, does it support Bitgo?

Thanks

**Comment by vitorgamer58** [comment]
Tokens ECR20 not supported, Refer to https://github.com/InfraexDev/peatio/issues/111 and https://github.com/InfraexDev/peatio/issues/102

**Comment by shouton** [comment]
Muhummud gave a good sample, and you can do it yourself ERC20 is not that difficult.

I did it without any problems, because you see ERC20 is sort of ETH.

**Comment by vitorgamer58** [comment]
yep, I was seeing a commit on this in another fork, but I do not know how I would do it on my fork since I saw some differences. https://github.com/rubykube/peatio/commit/aebf82f2c8923ada8444ce81588e3d56b46148b6

**Comment by appsdev9** [comment]
@shouton, may I know where I can find the Muhummud's ERC20 sample implementation?

**Comment by vitorgamer58** [comment]
hey Guys, look for this https://github.com/rubykube/peatio/issues/1373

**Comment by vitorgamer58** [comment]
https://github.com/rubykube/peatio/pull/913

**Comment by shouton** [comment]
@appsdev9 

are you a programmer? if you are, follow the sample of ETH, you can do it.
if not buy it.

**Comment by danclickomy** [comment]
@shouton Hey mate, that's awesome that you managed to do that. Can I ask you for some help on how to implement it myself?
